### PR TITLE
profiles/graphic_drivers: Add open nvidia modules profiles

### PR DESCRIPTION
**Pull request by ventureoo**

### CODE REVIEW

**Comment by ptr1337** [comment]
Tomorrow should come the 560 Driver into the CachyOS repo, as soon 6.10.4 drops.
Can you undraft it?

File: profiles/pci/graphic_drivers/profiles.toml [hunk 1]
```diff
@@ -16,25 +16,89 @@
 # Or set of devices depending on the generation:
 # device_name_pattern = '(AD)\w+'
 
-[nvidia-dkms.40xxcards]
-desc = 'Closed source NVIDIA drivers(40xx series) for Linux (Latest)'
-priority = 9
-device_name_pattern = '(AD)\w+'
+[nvidia-open-dkms]
+desc = 'Closed source NVIDIA drivers for Linux (Latest)'
```
Comment (vnepogodin) [inline profiles/pci/graphic_drivers/profiles.toml:20]: Rather open modules

File: profiles/pci/graphic_drivers/profiles.toml [hunk 2]
```diff
@@ -16,25 +16,89 @@
...
+"""
+post_remove = """
+    rm -f /etc/mkinitcpio.conf.d/10-chwd.conf
+    rm -f /etc/profile.d/nvidia-vaapi.sh
+    mkinitcpio -P
+"""
+device_name_pattern = "([A-Z]+[0-9]+[^M]*)[[:blank:]].*"
+
+[nvidia-open-dkms.prime]
+desc = 'Closed source NVIDIA drivers for Linux (Latest)'
```
Comment (vnepogodin) [inline profiles/pci/graphic_drivers/profiles.toml:62]: Same

**Comment by vnepogodin** [comment]
priority looks incorrect here.
higher priority value -> getting sorted as first in the Vec. therefore will get installed the profile which is the first to be found for the device in normal order from begin..end

**Comment by ventureoo** [comment]
> priority looks incorrect here. higher priority value -> getting sorted as first in the Vec. therefore will get installed the profile which is the first to be found for the device in normal order from begin..end

This issue is that the newest profile tends to have very generic matching conditions, like `device_ids = '*"`, so if we start there, that profile will most likely be selected first and all others will not be installed, even if we are dealing with a card that needs the 390xx driver. So we must first check all the "old" profiles and discard them via device_name_pattern, so that at the end we have the most suitable one selected.

**Review by vnepogodin — APPROVED** [review]
fine

**Pull request merged**